Testing/Tests/LinkedList_Test.cpp

```diff
@@ -48,34 +55,62 @@ TEST(LinkedList, CountIncreasesAfterPushFront)
    CHECK(1 == LinkedList_Count(&instance));
 }
 
-TEST(LinkedList, CountIncreasesAfterPushBack)
+TEST(LinkedList, CountIncreasesAfterPushFrontFiveNodes)
```

Comment:
Is there any specific reason for 5 nodes? If not a better test name might be `CountIncreasesAfterPushFrontMultipleNodes`

Same with the test name `CountIncreasesAfterPushBackSixNodes` -> `CountIncreasesAfterPushBackMultipleNodes`